eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/CertsSpec.hs

```diff
@@ -0,0 +1,87 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Test.Cardano.Ledger.Conway.Imp.CertsSpec (
+  spec,
+) where
```

Comment:
Spec modules rarely export anything but the `spec`, so there is no need to be git diff friendly here
```suggestion
module Test.Cardano.Ledger.Conway.Imp.CertsSpec (spec) where
```